### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by ArtemKa2208**
homework 1

### CODE REVIEW

File: mean.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let array = [14,20,5,7,83,12,11];
+function getMean(array){
```
Comment (StanislawVictorovich) [inline mean.js:3]: wrong indentations

File: sort.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+let array = [14,20,57,74,83,121,131];
+function isArraySort(array){
+      let currentValues = array[0]
+      for(let i = 1; i < array.length; i++){
+            if(currentValues < array[i]){
```
Comment (StanislawVictorovich) [inline sort.js]: need to rewrite if-else by the most simple construction. 1 return with 1 condition

File: mean.js (hunk 1 above)
Comment (ArtemKa2208) [reply mean.js:3]: done

File: sort.js (hunk 2 above)
Comment (ArtemKa2208) [reply sort.js]: done

**Review by StanislawVictorovich — APPROVED** [review]

**Pull request merged**